lib/bolt/config.rb

```diff
@@ -70,7 +75,29 @@ def default_paths
       [File.join(root_path, 'bolt.yaml'), File.join(root_path, 'bolt.yml')]
     end
 
+    def normalize_log(target)
+      return target if target == 'console'
+      target = target[5..-1] if target.start_with?('file:')
```

Comment:
Currently the only reason is probably creating a file named `console`, which would not be possible otherwise.
But going forward we may want to add more logging options (syslog maybe), in which case we'll probably wan't to introduce something like a log URI anyway so I wanted to make sure the file based logs are prepared for that.

Reply:
That makes sense.